### ISSUE TITLE
Future Feature: Ship activity log

### ISSUE THREAD

**Issue by msummers123** [msummers123/Movecraft-3#83]
Seperate log file for ship activity only. Also make it take a note of nearby crafts when a ship sinks, and its tonnage, pilot, etc

**Comment by mwkaicz** [comment]
maybe parse logs per world and date should be good (/logs/worldName/yyyy-mm-dd.log)

**Comment by TylerS1066** [comment]
I like mwkaicz's idea, it would make removing logs older than x days very simple (x being defined in the config)

**Comment by keough99** [comment]
Good idea and maybe movecraft had its own log file and its own log for collision explosions and piloted by who and location
